doc/reference/reference_lua/tap.rst

```diff
@@ -309,23 +309,32 @@ so on.
 
     .. _taptest-istype:
 
-    .. method:: isnil(value, test-name)
-                isstring(value, test-name)
-                isnumber(value, test-name)
-                istable(value, test-name)
-                isboolean(value, test-name)
-                isudata(value, test-name)
-                iscdata(value, test-name)
+    .. method:: isnil(test, v, message, extra)
+                isstring(test, v, message, extra)
+                isnumber(test, v, message, extra)
+                istable(test, v, message, extra)
+                isboolean(test, v, message, extra)
+                isudata(test, v, utype, message, extra)
+                iscdata(test, v, ctype, message, extra)
```

Comment:
- `test` param should be removed, it is passed implicitly
- replace `v` to `value` back (one symbol variables is less convenient)